Apps/Playground/CMakeLists.txt

```diff
@@ -1,15 +1,17 @@
 set(BABYLONSCRIPTS
-    "../BabylonScripts/babylonjs.loaders.js"
-    "../BabylonScripts/babylonjs.loaders.js.map"
-    "../BabylonScripts/babylon.max.js"
-    "../BabylonScripts/babylon.max.js.map"
-    "../BabylonScripts/babylonjs.materials.js"
-    "../BabylonScripts/babylonjs.materials.js.map"
-    "../BabylonScripts/babylon.gui.js"
-    "../BabylonScripts/babylon.gui.js.map"
-    "../BabylonScripts/ammo.js"
-    "../BabylonScripts/meshwriter.min.js"
-    "../BabylonScripts/recast.js")
+    "../node_modules/babylonjs-loaders/babylonjs.loaders.js"
```

Comment:
You also need to update these lines:

https://github.com/BabylonJS/BabylonNative/blob/master/Apps/Playground/Scripts/experience.js#L1-L4